### PR TITLE
feat(review): add a load more button

### LINKED ISSUE #184: Add a show more button in the history view
As a user, I can go back in the history of the builds

### Motivation

Something you might be looking for an old build

### Detail

Have a look at how travis is doing it: https://travis-ci.org/callemall/material-ui/builds.

### PR DESCRIPTION
**Pull request by oliviertassinari**
Needs #228.

Closes #184.

### CODE REVIEW

File: src/review/routes/repository/detailsEpic.js [hunk 1]
```diff
@@ -1,24 +1,22 @@
 import graphQLClient from 'modules/graphQL/client'
 import actionTypes from 'review/modules/redux/actionTypes'
 
+const first = process.env.NODE_ENV === 'production' ? 10 : 5
```
Comment (neoziro) [inline src/review/routes/repository/detailsEpic.js]: Why?

File: src/modules/components/WatchTask.js [hunk 2]
```diff
@@ -19,25 +19,22 @@ function renderInContainer(props, node) {
...
   }
 
-  if (state === PROGRESS) {
+  if (task.state === PROGRESS) {
     return renderInContainer(props, <CircularProgress />)
   }
 
-  if (state === SUCCESS) {
+  if (task.state === SUCCESS) {
     return children()
```
Comment (neoziro) [inline src/modules/components/WatchTask.js]: This component is not in the React way, I think we should transform it but for now good for me.

File: src/review/routes/repository/RepositoryDetails.js [hunk 3]
```diff
@@ -23,55 +24,55 @@ function RepositoryDetails(props) {
...
-                <Text>
-                  Repository not found
-                </Text>
-              </WatchTaskContainer>
-            )
-          }
+    <div>
+      <Paper className={classes.paper}>
+        <WatchTask task={fetch}>
+          {() => {
```
Comment (neoziro) [inline src/review/routes/repository/RepositoryDetails.js]: Crappy code, if if if

File: src/modules/components/WatchTask.js (hunk 2 above)
Comment (oliviertassinari) [inline src/modules/components/WatchTask.js]: It's more a matter of [convention](https://github.com/bvaughn/react-virtualized/blob/master/docs/usingAutoSizer.md#can-i-use-autosizer-within-a-flex-container)

File: src/review/routes/repository/detailsEpic.js (hunk 1 above)
Comment (oliviertassinari) [inline src/review/routes/repository/detailsEpic.js]: It's just for the seed. I'm gonna generate more data.

**Pull request merged**